src/map/zone.h

```diff
@@ -617,9 +616,9 @@ class CZone
     CBattlefieldHandler* m_BattlefieldHandler; // BCNM Instances in this zone
     CCampaignHandler*    m_CampaignHandler;    // WOTG campaign information for this zone
 
-    CNavMesh* m_navMesh;            // zones navmesh for finding paths
-    bool      m_updatedNavmesh;     // Flag to turn off special path code.
-    bool      m_zoneCarefulPathing; // Zonewide careful pathing, should only be used with updated meshes.
+    bool                                  m_updatedNavmesh;     // Flag to turn off special path code.
+    bool                                  m_zoneCarefulPathing; // Zonewide careful pathing, should only be used with updated meshes.
+    std::unique_ptr<CNavigationContainer> PNavigation;          // Holds dtNavMesh & queries
```

Comment:
Typedef this as something shorter, or align it one line below, so the formatting doesn't get wacky